### DISCUSSION TITLE
유니티 테스트 모드에서 맵 정원 수용 가능 여부

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#336] [5. Q&A]
안녕하세요.
유니티에서 Play on ZEPETO를 통해 생성한 월드에서
QR 코드를 통해 월드의 정원에 접속하는 것이 가능한가요?
16명 제한으로 설정했지만 그 미만으로 제한이 따로 있는 지 궁금합니다.
확인 부탁드립니다.
감사합니다.

![image](https://user-images.githubusercontent.com/5335151/152750696-df8b1306-fed9-4f47-9a22-879278dfee90.png)

**Comment by x4cf** [comment]
최대 입장가능 인원 수 설정 질문이신건가요?
보여주신 장면의 16이라는 숫자를 클릭하고 2~16사이의 숫자로 입력하면 방 최대 입장가능 인원 수가 변경됩니다 :)

**Reply by ohhagi** [reply]
혹시 맵 등록과 무관하게 PC 유니티를 통한 테스트 모드에서도 설정한 인원 수 만큼 입장이 가능한가요?

**Reply by x4cf** [reply]
네 가능합니다

**Comment by hyeonjin-jo** [comment]
안녕하세요.

위에서 잘 대답해 주신 것 같은데 한 번 더 정리를 해보면,

유니티 에디터에서 MaxClients 값 설정 후 빌드를 하고 월드를 등록하면 제페토 앱에서 접근하는 월드의 룸 최대 인원수가 해당 값으로 설정이 됩니다.

이후 유니티 에디터에서 MaxClients 값을 수정하고 빌드를 하지 않고 로컬테스트를 진행 한다면 변경된 값은 로컬에서만 적용이 됩니다.

추가로 MaxClients값 이외에도 lock, unlock기능을 활용해서 특정 상황에 룸입장을 제한할 수 있습니다.

https://studio.zepeto.me/guides/temporarily-restricting-room-access
